modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/ObjectEntryFolderDTOConverter.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.headless.delivery.internal.dto.v1_0.converter;
+
+import com.liferay.headless.delivery.dto.v1_0.ObjectEntryFolder;
+import com.liferay.headless.delivery.dto.v1_0.util.CreatorUtil;
+import com.liferay.object.service.ObjectEntryFolderLocalService;
+import com.liferay.object.service.ObjectEntryLocalService;
+import com.liferay.portal.kernel.model.Group;
+import com.liferay.portal.kernel.service.GroupLocalService;
+import com.liferay.portal.kernel.service.UserLocalService;
+import com.liferay.portal.kernel.util.Portal;
+import com.liferay.portal.vulcan.dto.converter.DTOConverter;
+import com.liferay.portal.vulcan.dto.converter.DTOConverterContext;
+import com.liferay.portal.vulcan.util.GroupUtil;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Alicia García
+ */
+@Component(
+	property = "dto.class.name=com.liferay.object.model.ObjectEntryFolder",
+	service = DTOConverter.class
+)
+public class ObjectEntryFolderDTOConverter
+	implements DTOConverter
+		<com.liferay.object.model.ObjectEntryFolder, ObjectEntryFolder> {
+
+	@Override
+	public String getContentType() {
+		return ObjectEntryFolder.class.getSimpleName();
+	}
+
+	@Override
+	public ObjectEntryFolder toDTO(DTOConverterContext dtoConverterContext)
+		throws Exception {
+
+		com.liferay.object.model.ObjectEntryFolder objectEntryFolder =
```

Comment:
@AliciaGarciaGarcia, about the naming....I already see in my mind that we are going to get a complaint....

Reply:
ANY suggestion?????


<img src="https://media3.giphy.com/media/v1.Y2lkPWJkM2VhNTdla2MzZmtrMmk1Z2dpNmlhYnppaGk0amphbDVqMWpremd0d255ejBmdiZlcD12MV9naWZzX3NlYXJjaCZjdD1n/K6WIhJ07gwGkIAQfwN/giphy.gif"/>

Reply:
seems we have an ObjectFolder in object-admin-rest......so.....seems it's valid.

Mapping to ObjectDefinition is also 1:1....

![](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExbHg3YXN3enEwZm5nYXhkM2I5Z2J6ajJ1cWZ5NWZ1dXpjYnp2MDkweiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/UQOtfc9uIXVv9IidCD/giphy.gif)

Reply:
@adolfopa, @AliciaGarciaGarcia, we can go with it, it's a common pattern in the object rest API